helm-charts/cubestore/templates/workers/statefulset.yaml

```diff
@@ -0,0 +1,104 @@
+{{- if gt (.Values.workers.workersCount | int) 0 }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ printf "%s-workers" (include "cubestore.fullname" .) }}
+  labels:
+    {{- include "cubestore.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.workers.workersCount }}
+  serviceName: {{ printf "%s-headless" (include "cubestore.fullname" .) }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: workers
+      {{- include "cubestore.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: workers
+        {{- include "cubestore.selectorLabels" . | nindent 8 }}
+    spec:
+      containers:
+        - name: cubestore
+          image: "cubejs/cubestore:v{{ .Chart.AppVersion }}"
+          ports:
+            - name: worker
+              containerPort: {{ .Values.workers.port }}
+              protocol: TCP
+          env:
+            {{- include "cubestore.common-env" . | nindent 12 }}
+            {{- $fullName := include "cubestore.fullname" . }}
+            {{- $headlessServiceName := printf "%s-%s" $fullName "headless" }}
+            {{- $workerPort := .Values.workers.port | int }}
+            - name: MY_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: CUBESTORE_SERVER_NAME
+              value: {{ printf "$(MY_POD_NAME).%s:%d" $headlessServiceName $workerPort}}
+            - name: CUBESTORE_WORKER_PORT
+              value: {{ .Values.workers.port | quote }}
+            {{- $workers := list }}
+            {{- range $e, $i := until ($.Values.workers.workersCount | int) }}
+            {{- $workers = append $workers (printf "%s-workers-%d.%s:%d" $fullName $i $headlessServiceName $workerPort) }}
+            {{- end }}
+            - name: CUBESTORE_WORKERS
+              value: {{ join "," $workers | quote }}
+            - name: CUBESTORE_META_ADDR
+              value: {{ printf "%s-router:%d" (include "cubestore.fullname" .) (.Values.router.metaPort | int) }}
+            - name: CUBESTORE_DATA_DIR
+              value: /cube/.cubestore/data
```

Comment:
@lvauvillier why use local mount path if the preferred method is the use of S3/GCP buckets? I'd make it "if not cloudprovider then" use local path (basically same as https://github.com/cube-js/cube.js/pull/3277/commits/793cc2db48dd728d17b5517feb6dbc95032e93e2#r691975909 )